scripts/generate_api_documentation.py

```diff
@@ -216,10 +215,9 @@ def generate_api_docs() -> None:
     """Generate the api docs."""
     shutil.rmtree(API_DIR, ignore_errors=True)
     API_DIR.mkdir()
-    with ThreadPoolExecutor() as executor:
-        _generate_apidocs_packages(executor)
-        _generate_apidocs_aea_modules(executor)
-        _generate_apidocs_plugins(executor)
```

Comment:
@Adamantios Running this script in a thread pool lead to the flaky issues. For now I've removed the thread pool usage. It'll be a bit slow but we won't have any issues.